module/chunks/chunkVerifier_test.go

```diff
@@ -246,7 +252,7 @@ func (s *ChunkVerifierTestSuite) TestEventsMismatch() {
 // TestServiceEventsMismatch tests verification behavior in case
 // of emitted service events not matching chunks'
 func (s *ChunkVerifierTestSuite) TestServiceEventsMismatch() {
```

Comment:
For clarity, I would suggest to rename a couple of the existing tests, to make the conceptual difference more explicit:
```suggestion
func (s *ChunkVerifierTestSuite) TestServiceEventsMismatch_SystemChunk() {
```